implementations/rust/ockam/ockam_command/src/enroll.rs

```diff
@@ -357,3 +360,27 @@ impl Auth0Service {
         Ok(())
     }
 }
+
+async fn update_enrolled_identity(
+    ctx: &Context,
+    opts: &CommandGlobalOpts,
+    node_name: &str,
+) -> Result<()> {
+    let mut identities = opts.state.identities.list()?;
+    identities.push(opts.state.identities.default()?);
```

Comment:
Why is this needed? Shouldn't the default identity should be returned by `list()`?

Reply:
Thank you for pointing that out, I've checked it again and it is not needed.

The locations where two different directories so I had assumed it is two different files, but the default identity is just a symlink to the identities directory.